turbo/generators/config.ts

```diff
@@ -51,30 +58,41 @@ export default function generator(plop) {
         path: "packages/{{ name }}/package.json",
         async transform(content, answers) {
           const pkg = JSON.parse(content);
+          // Add dependencies
```

Comment:
Since `answers` is `any` in this scope, we might wanna validate `answers`, like we do elsewhere:
```ts
if ("deps" in answers && typeof answers.deps === "string") {
  //...
}
```

Reply:
Good catch, amended!